README.md

```diff
@@ -58,15 +43,13 @@ Clone the FCCSW repository: **replace \<username\> by your github username**
 
```

Comment:
Then I guess the easiest solution is to redefine 
"export FCCSW=$FCCBASE/FCCSW" 
to
"export FCCSW=$PWD/FCCSW"